script.js

```diff
@@ -1,4 +1,383 @@
+var generateDeck = function (input) {
```

Comment:
I like how you tried a different method to generate the deck rather than used the standard code.